packages/core-js/internals/object-create.js

```diff
@@ -26,13 +37,27 @@ var NullProtoObject = function () {
   iframe.src = String(JS);
   iframeDocument = iframe.contentWindow.document;
   iframeDocument.open();
-  iframeDocument.write(LT + SCRIPT + GT + 'document.F=Object' + LT + '/' + SCRIPT + GT);
+  iframeDocument.write(scriptTag('document.F=Object'));
   iframeDocument.close();
-  NullProtoObject = iframeDocument.F;
-  while (length--) delete NullProtoObject[PROTOTYPE][enumBugKeys[length]];
-  return NullProtoObject();
+  return iframeDocument.F;
 };
 
+// Check for document.domain and active x support
+// No need to use active x approach when document.domain is not set
+// see https://github.com/es-shims/es5-shim/issues/150
+// variation of https://github.com/kitcambridge/es5-shim/commit/4f738ac066346
+var activeXDocument;
+var NullProtoObject = Object.create /* avoid initialization */ || (function () {
+  try {
+    /* global ActiveXObject */
+    activeXDocument = document.domain && new ActiveXObject('htmlfile');
+  } catch (error) { /* ignore */ }
+  var ProtoObject = activeXDocument ? NullProtoObjectViaActiveX(activeXDocument) : NullProtoObjectViaIFrame();
+  var length = enumBugKeys.length;
+  while (length--) delete ProtoObject[PROTOTYPE][enumBugKeys[length]];
+  return ProtoObject;
+})();
```

Comment:
It would be better to use delayed initialization like it was before, like:
```js
var NullProtoObject = function () {
  // ...
  NullProtoObject = something;
  return NullProtoObject();
}
```